devito/grid.py

```diff
@@ -201,3 +201,28 @@ def __setstate__(self, state):
         for k, v in state.items():
             setattr(self, k, v)
         self._distributor = Distributor(self.shape, self.dimensions)
+
+
+def staggered(function):
+    """
+    For a given function and it's staggered property `staggered`
+    returns a tuple of 0 or 1 for each dimension:
+    0 if not staggered in the dimension
+    1 if staggered in the dimension
+    """
+    s = (lambda x: x if type(x) is tuple else (x,))(function.staggered)
+    if function.staggered in ['node', None]:
```

Comment:
'node' and 'cell' should be defined through ``Tag``. ``git grep Tag`` for some examples.